tests/pkg_relic/tests/01-run.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Freie Universität Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import sys
+from testrunner import run
+
+
+def testfunc(child):
+    child.expect('OK \(\d+ tests\)', timeout=120)
```

Comment:
`flake8` is correctly complaining about this one, this should be a `regex` string 
```suggestion
    child.expect(r'OK \(\d+ tests\)', timeout=120)
```

When rebasing to `riot/master` it will not be detected anymore as it will be using `riotdocker` image that uses the repository `flake8` version but the error is still valid.